### ISSUE TITLE
gvisor/pkg/buffer: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by hussainmohd-a** [celzero/firestack#74]
```jsx
I ns: dispatch: fds: stop: eve(290) tun(279); err? <nil>
I proxy: exit: dial(tcp) to 31.13.79.2:443; err? <nil>
I tcp: new conn 1c16cd09a06f03c2 via proxy(Exit); src(10.111.222.1:48486) -> dst(31.13.79.2:443) for 10268
I rdial: write: first?(true) [100.71.97.44:45810->31.13.79.2:443] 382; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 31.13.84.2:443; err? <nil>
I tcp: new conn 512b53583e654720 via proxy(Exit); src(10.111.222.1:44182) -> dst(31.13.84.2:443) for 10268
I proxy: exit: dial(tcp) to [2a03:2880:f27b:2e2:face:b00c:0:6e2e]:5222; err? <nil>
I tcp: new conn 582ff8a766ee3a4a via proxy(Exit); src([fd66:f83a:c650::1]:38394) -> dst([2a03:2880:f27b:2e2:face:b00c:0:6e2e]:5222) for 10268
I rdial: write: first?(true) [100.71.97.44:48030->31.13.84.2:443] 2048; 1st write-err? <nil>
I rdial: write: first?(true) [100.71.97.44:48030->31.13.84.2:443] 229; 1st write-err? <nil>
I proxy: exit: dial(tcp) to [2a03:2880:f108:80:face:b00c:0:1ea0]:443; err? <nil>
I tcp: new conn 49d49106ba84a17f via proxy(Exit); src(10.111.222.1:47304) -> dst([2a03:2880:f108:80:face:b00c:0:1ea0]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:39338->[2a03:2880:f108:80:face:b00c:0:1ea0]:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to [2a03:2880:f08e:18:face:b00c:0:6206]:443; err? <nil>
I tcp: new conn 0ffc21fbc7f8d7a9 via proxy(Exit); src(10.111.222.1:48500) -> dst([2a03:2880:f08e:18:face:b00c:0:6206]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:47728->[2a03:2880:f08e:18:face:b00c:0:6206]:443] 382; 1st write-err? <nil>
E rdial: read: [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:47728<-[2a03:2880:f08e:18:face:b00c:0:6206]:443] 0; mustretry? true; err: EOF
I proxy: exit: dial(tcp) to [2a03:2880:f00c:108:face:b00c:0:6206]:443; err? <nil>
I tcp: new conn 603174f4281b2d5c via proxy(Exit); src(10.111.222.1:48508) -> dst([2a03:2880:f00c:108:face:b00c:0:6206]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443] 382; 1st write-err? <nil>
E rdial: read: [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296<-[2a03:2880:f00c:108:face:b00c:0:6206]:443] 0; mustretry? true; err: EOF
W rdial: readfrom: sz: 4989; err: readfrom tcp [2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443: write tcp [2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443: write: broken pipe
I proxy: exit: dial(tcp) to 142.251.175.188:5228; err? <nil>
I tcp: new conn 4db404a60ec8b9c9 via proxy(Exit); src(10.111.222.1:45850) -> dst(142.251.175.188:5228) for 10133
E c.upload: b85e6888d0a12280 [11] runtime error: invalid memory address or nil pointer dereference

github.com/celzero/firestack/intra/log.(*simpleLogger).Stack(0x40000dc0e0, 0x0, {0x400807af50?, 0x400807ae00?}, {0x400a4e2000, 0x10000, 0x10000})
	/home/jitpack/build/intra/log/logger.go:400 +0x2c0
github.com/celzero/firestack/intra/log.C({0x400807ae00, 0x61}, {0x400a4e2000, 0x10000, 0x10000})
	/home/jitpack/build/intra/log/log.go:136 +0xf8
github.com/celzero/firestack/intra/core.trace(0x40000dc0e0?, {0x400807ae00, 0x61})
	/home/jitpack/build/intra/core/dontpanic.go:101 +0x1a4
github.com/celzero/firestack/intra/core.Recover(0xb, {0x4007f8cce0, 0x1a})
	/home/jitpack/build/intra/core/dontpanic.go:74 +0x13c
panic({0x709856ca00?, 0x7098a4dfa0?})
	/home/jitpack/golang/go/src/runtime/panic.go:770 +0x124
gvisor.dev/gvisor/pkg/buffer.(*chunk).DecRef(...)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/buffer/chunk.go:106
gvisor.dev/gvisor/pkg/buffer.(*View).Release(0x4009490d20)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/buffer/view.go:107 +0x38
gvisor.dev/gvisor/pkg/buffer.(*Buffer).removeView(0x70980e91f0?, 0x4009490d20)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/buffer/buffer.go:37 +0x2c
gvisor.dev/gvisor/pkg/buffer.(*Buffer).Release(0x400a5957c8)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/buffer/buffer.go:73 +0x2c
gvisor.dev/gvisor/pkg/tcpip/transport/udp.(*endpoint).Read.deferwrap1.(*PacketBuffer).DecRef.1()
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/stack/packet_buffer.go:204 +0x3c
gvisor.dev/gvisor/pkg/tcpip/stack.(*packetBufferRefs).DecRef(0x400842f498?, 0x400842f4c8)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/stack/packet_buffer_refs.go:133 +0x70
gvisor.dev/gvisor/pkg/tcpip/stack.(*PacketBuffer).DecRef(...)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/stack/packet_buffer.go:199
gvisor.dev/gvisor/pkg/tcpip/transport/udp.(*endpoint).Read(_, {_, _}, {_, _, _})
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/transport/udp/endpoint.go:299 +0x49c
gvisor.dev/gvisor/pkg/tcpip/adapters/gonet.commonRead({0x4009f36000, 0x10000, 0x10000}, {0x7098661e48, 0x40088be008}, 0x4009198600, 0x400a1958c0, 0x400842fc98, {0x7098650000, 0x40083ece40})
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/adapters/gonet/gonet.go:311 +0xfc
gvisor.dev/gvisor/pkg/tcpip/adapters/gonet.(*UDPConn).ReadFrom(0x40083ece40, {0x4009f36000, 0x10000, 0x10000})
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/adapters/gonet/gonet.go:639 +0x74
gvisor.dev/gvisor/pkg/tcpip/adapters/gonet.(*UDPConn).Read(...)
	/tmp/gomobile-work-453886779/pkg/mod/gvisor.dev/gvisor@v0.0.0-20240629121841-891b40cf7fe0/pkg/tcpip/adapters/gonet/gonet.go:630
github.com/celzero/firestack/intra/netstack.(*GUDPConn).Read(0x400a37ce30?, {0x4009f36000?, 0x4dc?, 0x10000?})
	/home/jitpack/build/intra/netstack/udp.go:188 +0x8c
io.copyBuffer({0x7446d7c278, 0x400a37ce30}, {0x7446d7c298, 0x4008dcaea0}, {0x4009f36000, 0x10000, 0x10000})
	/home/jitpack/golang/go/src/io/io.go:429 +0x18c
io.CopyBuffer({0x7446d7c278?, 0x400a37ce30?}, {0x7446d7c298?, 0x4008dcaea0?}, {0x4009f36000?, 0x7098651b40?, 0x7098651b40?})
	/home/jitpack/golang/go/src/io/io.go:402 +0x38
github.com/celzero/firestack/intra/core.Pipe({0x7446d7c278, 0x400a37ce30}, {0x7446d7c298, 0x4008dcaea0})
	/home/jitpack/build/intra/core/cp.go:36 +0x1cc
github.com/celzero/firestack/intra.upload({0x4009966550, 0x10}, {0x709865baa0, 0x4008dcaea0}, {0x709865baf8, 0x400a37ce30}, 0x40092a20c0)
	/home/jitpack/build/intra/common.go:39 +0x10c
created by github.com/celzero/firestack/intra.forward in goroutine 189043
	/home/jitpack/build/intra/common.go:67 +0xe4
```

**Comment by hussainmohd-a** [comment]
```jsx
25 July 2024, 22:14:07: I udp: 25e23783d54036de (proxy? Exit@127.0.0.127:1337) 192.168.0.144:45946 -> 103.194.241.225:3478/103.194.241.225:3478 for uid 10268
W intra: makeIPPorts(v4? true, v6? false): all: 1; out: 163.70.139.128:3478
I proxy: exit: dial(udp) to 163.70.139.128:3478; err? <nil>
I udp: b63b978b8c8d7348 (proxy? Exit@127.0.0.127:1337) 192.168.0.144:40892 -> 163.70.139.128:3478/163.70.139.128:3478 for uid 10268
W intra: makeIPPorts(v4? true, v6? false): all: 1; out: 157.240.192.128:3478
I proxy: exit: dial(udp) to 157.240.192.128:3478; err? <nil>
I udp: 0b16156fe9e1612b (proxy? Exit@127.0.0.127:1337) 192.168.0.144:43882 -> 157.240.192.128:3478/157.240.192.128:3478 for uid 10268
W intra: makeIPPorts(v4? true, v6? false): all: 1; out: 157.240.23.128:443
I proxy: exit: dial(udp) to 157.240.23.128:443; err? <nil>
I udp: b85e6888d0a12280 (proxy? Exit@127.0.0.127:1337) 192.168.0.144:40058 -> 157.240.23.128:443/157.240.23.128:443 for uid 10268
I tcp: gconn 108f24caa8f3661e firewalled from 10.111.222.1:49988 -> 216.239.36.158:443 (dom: instantmessaging-pa-jms-ap.googleapis.com + / real: 216.239.36.158) for 10129; stall? 0s
I proxy: exit: dial(udp) to 103.194.241.228:443; err? <nil>
I udp: effdde398a11e57c (proxy? Exit@127.0.0.127:1337) 192.168.0.144:48944 -> 103.194.241.228:443/103.194.241.228:443 for uid 10268
W rdial: readfrom: sz: 3381; err: readfrom tcp 192.168.0.144:48536->142.251.37.234:443: write tcp 192.168.0.144:48536->142.251.37.234:443: write: broken pipe
I proxy: base: dial(tcp) to sky.rethinkdns.com:443; err? <nil>
I proxy: exit: dial(tcp) to 13.37.18.215:443; err? <nil>
I tcp: new conn 2e06fb34b6f6e5e6 via proxy(Exit); src(10.111.222.1:41346) -> dst(13.37.18.215:443) for 10166
I rdial: write: first?(true) [192.168.0.144:38470->13.37.18.215:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 13.37.18.215:443; err? <nil>
I tcp: new conn d6f6b65ef8662f90 via proxy(Exit); src(10.111.222.1:41342) -> dst(13.37.18.215:443) for 10166
I rdial: write: first?(true) [192.168.0.144:38462->13.37.18.215:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 13.36.49.3:443; err? <nil>
I tcp: new conn cb618824fe33f4f2 via proxy(Exit); src(10.111.222.1:41356) -> dst(13.36.49.3:443) for 10166
I rdial: write: first?(true) [192.168.0.144:41928->13.36.49.3:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 52.47.118.143:443; err? <nil>
I tcp: new conn e0d6ed11e47bc968 via proxy(Exit); src(10.111.222.1:41362) -> dst(52.47.118.143:443) for 10166
I rdial: write: first?(true) [192.168.0.144:44622->52.47.118.143:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 13.37.18.215:443; err? <nil>
I tcp: new conn 07f280186e00ab83 via proxy(Exit); src(10.111.222.1:41370) -> dst(13.37.18.215:443) for 10166
I rdial: write: first?(true) [192.168.0.144:38472->13.37.18.215:443] 517; 1st write-err? <nil>
I cache: del: 0; ref: 0; tot: 7 / high? false
E rdial: read: [192.168.0.144:38470<-13.37.18.215:443] 0; mustretry? true; err: EOF
E rdial: read: [192.168.0.144:41928<-13.36.49.3:443] 0; mustretry? true; err: EOF
E rdial: read: [192.168.0.144:44622<-52.47.118.143:443] 0; mustretry? true; err: EOF
E rdial: read: [192.168.0.144:38472<-13.37.18.215:443] 0; mustretry? true; err: EOF
I proxy: exit: dial(tcp) to 163.70.128.61:443; err? <nil>
I tcp: new conn 52d0599141738dcd via proxy(Exit); src(10.111.222.1:42292) -> dst(163.70.128.61:443) for 10259
I rdial: write: first?(true) [192.168.0.144:48172->163.70.128.61:443] 327; 1st write-err? <nil>
W intra: makeIPPorts(v4? true, v6? false): all: 1; out: 103.194.241.227:443
I proxy: exit: dial(tcp) to 103.194.241.227:443; err? <nil>
I tcp: new conn 76b9ad26ca3db75a via proxy(Exit); src(10.111.222.1:47846) -> dst(103.194.241.227:443) for 10259
I rdial: write: first?(true) [192.168.0.144:45228->103.194.241.227:443] 1121; 1st write-err? <nil>
W rdial: readfrom: sz: 591; err: readfrom tcp 192.168.0.144:41730->74.125.24.95:443: read tcp 142.250.192.138:443: connection reset by peer
W rdial: readfrom: sz: 646; err: readfrom tcp 192.168.0.144:39122->74.125.200.95:443: read tcp 142.250.192.138:443: connection reset by peer
W rdial: readfrom: sz: 678; err: readfrom tcp 192.168.0.144:45470->142.251.175.95:443: read tcp 142.250.192.138:443: connection reset by peer
W rdial: readfrom: sz: 1137; err: readfrom tcp 192.168.0.144:47464->142.250.71.106:443: read tcp 142.250.192.138:443: connection reset by peer
W rdial: readfrom: sz: 1095; err: readfrom tcp 192.168.0.144:43656->142.251.12.94:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 1073; err: readfrom tcp 192.168.0.144:39332->142.250.183.42:443: read tcp 142.250.192.138:443: connection reset by peer
W rdial: readfrom: sz: 609; err: readfrom tcp 192.168.0.144:39968->216.58.205.35:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 1031; err: readfrom tcp 192.168.0.144:37502->142.250.179.99:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 641; err: readfrom tcp 192.168.0.144:39964->216.58.205.35:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 673; err: readfrom tcp 192.168.0.144:43298->74.125.130.94:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 591; err: readfrom tcp 192.168.0.144:37508->142.250.179.99:443: read tcp 142.250.182.195:443: connection reset by peer
W rdial: readfrom: sz: 614; err: readfrom tcp 192.168.0.144:37538->142.250.70.74:443: read tcp 142.250.192.138:443: connection reset by peer
E ns: tcp: forwarder: synack(complete? true) src(10.111.222.1:44448) => dst(216.239.34.223:443); err(connection was refused)
E tcp: 224c2b05410af743 connect err connection was refused; 10.111.222.1:44448 -> 216.239.34.223:443 for 10289
I dnsx: bootstrap! system:53 -> [117.96.122.78 117.96.122.34 2401:4900:50:9::230 2401:4900:50:9::186 192.168.0.1]
I dns53: (System) pre-resolved system:53 to 117.96.122.78,117.96.122.34,2401:4900:50:9::230,2401:4900:50:9::186,192.168.0.1; ok? true
I dns53: (System) setup: system:53; pre-ips? true; relay? false
I cache: (CacheSystem) setup: cached.system.system:53; opts: ttl=10m0s;bumps=10;size=256
I dns: add transport System@system.system:53; cache? true
I dns: new system dns from 117.96.122.78,117.96.122.34,2401:4900:50:9::230,2401:4900:50:9::186,192.168.0.1; ok? true
W dns64: overwrite existing ip64([]) for resolver(__underlay)
I dns: default: DNS reinit  rethink w/ 117.96.122.78,117.96.122.34,2401:4900:50:9::230,2401:4900:50:9::186,192.168.0.1
I dnsx: bootstrap! rethink:53 -> [117.96.122.78 117.96.122.34 2401:4900:50:9::230 2401:4900:50:9::186 192.168.0.1]
I dns53: (Bootstrap) pre-resolved rethink:53 to 117.96.122.78,117.96.122.34,2401:4900:50:9::230,2401:4900:50:9::186,192.168.0.1; ok? true
I dns53: (Bootstrap) setup: rethink:53; pre-ips? true; relay? false
I dns: default: removing DNS rethink[d.rethink.rethink:53]
I dns: default: start; DNS with rethink[d.rethink.rethink:53]; ok? true
I rdial: commondial: ip 117.96.122.34 works for system:53
I proxy: exit: dial(udp) to system:53; err? <nil>
W rdial: readfrom: sz: 2981; err: readfrom tcp 192.168.0.144:45776->172.253.118.17:443: read tcp 142.250.183.101:443: connection reset by peer
W rdial: readfrom: sz: 5825; err: readfrom tcp 192.168.0.144:44614->64.233.170.83:443: read tcp 172.217.20.165:443: connection reset by peer
W rdial: readfrom: sz: 17685; err: readfrom tcp 192.168.0.144:39716->31.13.84.9:443: read tcp 163.70.128.19:443: connection reset by peer
W rdial: readfrom: sz: 2748; err: readfrom tcp 192.168.0.144:45540->74.125.200.95:443: read tcp 216.58.215.42:443: connection reset by peer
W rdial: readfrom: sz: 731; err: readfrom tcp 192.168.0.144:43054->91.108.56.106:443: read tcp 91.108.56.106:443: connection reset by peer
W rdial: readfrom: sz: 756; err: readfrom tcp 192.168.0.144:39372->13.248.212.111:443: read tcp 13.248.212.111:443: connection reset by peer
W rdial: readfrom: sz: 10856; err: readfrom tcp 192.168.0.144:47772->13.107.42.14:443: read tcp 13.107.42.14:443: connection reset by peer
W rdial: readfrom: sz: 2746; err: readfrom tcp 192.168.0.144:38384->157.240.15.10:443: read tcp 31.13.79.2:443: connection reset by peer
W rdial: readfrom: sz: 1040; err: readfrom tcp 192.168.0.144:39104->157.240.16.32:443: read tcp 163.70.128.34:443: connection reset by peer
W rdial: readfrom: sz: 833; err: readfrom tcp 192.168.0.144:42060->76.223.92.165:443: read tcp 13.248.212.111:443: connection reset by peer
W rdial: readfrom: sz: 5858; err: readfrom tcp 192.168.0.144:38880->57.144.120.144:443: read tcp 163.70.143.16:443: connection reset by peer
W rdial: readfrom: sz: 1629; err: readfrom tcp 192.168.0.144:38462->13.37.18.215:443: read tcp 13.37.18.215:443: connection reset by peer
W rdial: readfrom: sz: 2671; err: readfrom tcp 192.168.0.144:49390->52.46.131.184:443: read tcp 52.119.198.118:443: connection reset by peer
W rdial: readfrom: sz: 2247; err: readfrom tcp 192.168.0.144:41740->216.239.32.223:443: read tcp 216.239.34.223:443: connection reset by peer
W rdial: readfrom: sz: 1205; err: readfrom tcp 192.168.0.144:45228->103.194.241.227:443: read tcp 103.194.241.227:443: connection reset by peer
W rdial: readfrom: sz: 2931; err: readfrom tcp 192.168.0.144:48172->163.70.128.61:443: read tcp 157.240.235.61:443: connection reset by peer
I ns: dispatch: swap: tun(279 => 345); err <nil>
I tun: new link; fd(345), mtu(1500); err? <nil>
I ns: tun(279): wrapup: immediate (loopback)
W rdial: readfrom: sz: 10667; err: readfrom tcp 192.168.0.144:37844->157.240.16.2:443: read tcp 163.70.128.19:443: connection reset by peer
I dialers: ips: protos set to 46; diff? true
I mdns: setup: 46
I cache: (Cachemdns) setup: cached.224.0.0.251:5353; opts: ttl=10m0s;bumps=10;size=256
I dns: add transport mdns@224.0.0.251:5353; cache? true
I ns: tun(345): dispatchLoop: start
W rdial: readfrom: sz: 1389; err: readfrom tcp 192.168.0.144:44298->142.250.67.234:443: read tcp 142.251.42.106:443: connection reset by peer
W dns64: udp: could not query __underlay or empty ans; err <nil>
I proxy: exit: dial(udp) to 8.8.8.8:53; err? <nil>
I udp: f739af79412f8cf7 (proxy? Exit@127.0.0.127:1337) 100.71.97.44:43805 -> 8.8.8.8:53/8.8.8.8:53 for uid 10259
I proxy: refresh all 3
I proxy: refreshed 3 / 3: [Exit Base Block]
I ns: dispatch: fds: stop: eve(290) tun(279); err? <nil>
I proxy: exit: dial(tcp) to 31.13.79.2:443; err? <nil>
I tcp: new conn 1c16cd09a06f03c2 via proxy(Exit); src(10.111.222.1:48486) -> dst(31.13.79.2:443) for 10268
I rdial: write: first?(true) [100.71.97.44:45810->31.13.79.2:443] 382; 1st write-err? <nil>
I proxy: exit: dial(tcp) to 31.13.84.2:443; err? <nil>
I tcp: new conn 512b53583e654720 via proxy(Exit); src(10.111.222.1:44182) -> dst(31.13.84.2:443) for 10268
I proxy: exit: dial(tcp) to [2a03:2880:f27b:2e2:face:b00c:0:6e2e]:5222; err? <nil>
I tcp: new conn 582ff8a766ee3a4a via proxy(Exit); src([fd66:f83a:c650::1]:38394) -> dst([2a03:2880:f27b:2e2:face:b00c:0:6e2e]:5222) for 10268
I rdial: write: first?(true) [100.71.97.44:48030->31.13.84.2:443] 2048; 1st write-err? <nil>
I rdial: write: first?(true) [100.71.97.44:48030->31.13.84.2:443] 229; 1st write-err? <nil>
I proxy: exit: dial(tcp) to [2a03:2880:f108:80:face:b00c:0:1ea0]:443; err? <nil>
I tcp: new conn 49d49106ba84a17f via proxy(Exit); src(10.111.222.1:47304) -> dst([2a03:2880:f108:80:face:b00c:0:1ea0]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:39338->[2a03:2880:f108:80:face:b00c:0:1ea0]:443] 517; 1st write-err? <nil>
I proxy: exit: dial(tcp) to [2a03:2880:f08e:18:face:b00c:0:6206]:443; err? <nil>
I tcp: new conn 0ffc21fbc7f8d7a9 via proxy(Exit); src(10.111.222.1:48500) -> dst([2a03:2880:f08e:18:face:b00c:0:6206]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:47728->[2a03:2880:f08e:18:face:b00c:0:6206]:443] 382; 1st write-err? <nil>
E rdial: read: [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:47728<-[2a03:2880:f08e:18:face:b00c:0:6206]:443] 0; mustretry? true; err: EOF
I proxy: exit: dial(tcp) to [2a03:2880:f00c:108:face:b00c:0:6206]:443; err? <nil>
I tcp: new conn 603174f4281b2d5c via proxy(Exit); src(10.111.222.1:48508) -> dst([2a03:2880:f00c:108:face:b00c:0:6206]:443) for 10268
I rdial: write: first?(true) [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443] 382; 1st write-err? <nil>
E rdial: read: [[2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296<-[2a03:2880:f00c:108:face:b00c:0:6206]:443] 0; mustretry? true; err: EOF
W rdial: readfrom: sz: 4989; err: readfrom tcp [2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443: write tcp [2401:4900:6275:f46f:5090:1cff:feb3:d4b8]:38296->[2a03:2880:f00c:108:face:b00c:0:6206]:443: write: broken pipe
I proxy: exit: dial(tcp) to 142.251.175.188:5228; err? <nil>
I tcp: new conn 4db404a60ec8b9c9 via proxy(Exit); src(10.111.222.1:45850) -> dst(142.251.175.188:5228) for 10133
E c.upload: b85e6888d0a12280 [11] runtime error: invalid memory address or nil pointer dereference
```

**Comment by ignoramous** [comment]
Latest gvisor presumably has fixed this. https://github.com/google/gvisor/pull/10958
